index.js

```diff
@@ -1,19 +1,15 @@
+const stripIgnoredCharacters = require("graphql/utilities")
+  .stripIgnoredCharacters;
+
 module.exports = function(source) {
   var imports = [];
 
-  var minified = source
-    .replace(/^#import.*/gm, matched => {
-      imports.push(matched);
-      return "";
-    })
-    .replace(/#.*/g, "")
-    .replace(/\s(?![a-zA-Z_\.])/g, "")
-    .replace(/(?<![\(\,\:])\$/g, ",$")
-    .replace(/:\s+/g, ":")
-    .replace(/,\s+/g, ",")
-    .replace(/\{\s?/g, "{")
-    .replace(/\}\s?/g, "}")
-    .trim();
+  var sourceWithoutImports = source.replace(/^#import.*/gm, matched => {
+    imports.push(matched);
+    return "";
+  });
+
+  var minified = stripIgnoredCharacters(sourceWithoutImports);
 
-  return [...imports, minified].join("\n");
+  return imports.concat(minified).join("\n");
```

Comment:
Why to change this?

Reply:
hmm, I’m actually not sure. I’ll revert.

Reply:
Taking advantage of this, please use this guide to the commit message:
http://karma-runner.github.io/1.0/dev/git-commit-msg.html

Reply:
can do. would this be a `refactor` or ` feat`?

Reply:
I thinks it's a `refactor`

Reply:
not sure what to put in the parens either. How’s this:

```
refactor: Replace regex with stripIgnoredCharacters call
```

According to the karma commit message guidelines you can omit the parens "if the change is a global or difficult to assign to a single component". I guess this would be global?

Reply:
Yeah, can be 😉

Reply:
done!

Reply:
It should be the commit message, not PR's title 😅 
Then I can do `rebase and merge`

Reply:
@gabsprates ah, sorry I missed this. the rebase UI in GitHub should let you customise the commit message for the squashed commits (or in this case, _commit_). I believe it defaults to the name of the PR, which is why I changed it there.